.scripts/buildDB.sh

```diff
@@ -1,3 +1,3 @@
 sqlite3 identity.sqlite \
-"CREATE TABLE account(uid INTEGER PRIMARY KEY, username VARCHAR(30) NOT NULL UNIQUE, first_name VARCHAR(10) NOT NULL, last_name VARCHAR(10) NOT NULL, email VARCHAR(256) NOT NULL, email_status TINYINT NOT NULL, password_hash VARBINARY(64) NOT NULL, password_salt VARBINARY(64) NOT NULL, create_ts BIGINT NOT NULL, update_ts BIGINT NOT NULL, description VARCHAR(500) NOT NULL, block_until_ts BIGINT NOT NULL, consecutive_fails INTEGER NOT NULL DEFAULT 0);"
+"CREATE TABLE account(uid INTEGER PRIMARY KEY, username VARCHAR(30) NOT NULL UNIQUE, first_name VARCHAR(10) NOT NULL, last_name VARCHAR(10) NOT NULL, email VARCHAR(256) NOT NULL, email_status TINYINT NOT NULL, password_hash VARBINARY(64) NOT NULL, password_salt VARBINARY(64) NOT NULL, create_ts BIGINT NOT NULL, update_ts BIGINT NOT NULL, description VARCHAR(500) NOT NULL, block_until_ts BIGINT NOT NULL DEFAULT 1561761368763, consecutive_fails INTEGER NOT NULL DEFAULT 0);"
```

Comment:
Because we have removed `setBlockUntilTime()` and `setConsecutiveFails()` from both `SqlAccountCreate` and `SqlAccountUpdate`, as from a previous PR we talked about not that we shouldn't have these interfaces to allow callers to control the block until time/consecutive fails.
So what we did instead is: 
- for `SqlAccountCreate`: set default values in DB for these two fields and `SqlAccountCreate` has nothing to do with those two fields whatsoever.
- for `SqlAccountUpdate`: we just used the accountModel member field to do the updates to these two fields while not exposing to the user.

The current default value for the blockUntil field is just a past time, so that it is just easier to do time comparison than to a `null`.

Reply:
I think we probably could remove the `DEFAULT` and set the two fields implicitly in `create()` of `SqlAccountCreate` so that they are not exposed to the upper level.

Reply:
Addressed.